### ISSUE TITLE
When does the SRS error `API_TOKEN_ALREADY_USED` come up?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-ios#5042]
## Avoid duplicates
* [X] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Question is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Question has not already been asked in another issue

## Your Question
Hey there! I have a user on Twitter (https://twitter.com/frosch59/status/1623408635228430338?s=20&t=CgHV-jQ8lNyyhewkP7YXYA) who sees the `API_TOKEN_ALREADY_USED` error. When does the app display this error? 

---
Internal Tracking ID: [EXPOSUREAPP-14714](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14714)

**Comment by thomasaugsten** [comment]
It is very likely that the user deleted and installed the app. But the token to submit self test can only created once by month and will be deleted with the app deletion

**Comment by Ein-Tim** [comment]
@thomasaugsten 

Mh, strange. From what I can tell what the user wrote he updated to version 3.0.1 and got the error. I will try to get more information from the user and relay it here.

**Comment by Ein-Tim** [comment]
Okay, so I contacted the user via DM and got the following information:
"I have updated from 2.28 [to version 3.0.1].  Yes, the certificates are just gone. The recycle bin is empty."

It seems like there is a (race) condition which leads to a deletion of all certificates.

@thomasaugsten Let me know if you want more information to investigate this further. Is there anything the user can do to still warn using the SRS feature, as the user is currently positive?
